### PR TITLE
Added Swagger UI

### PR DESCRIPTION
**Pull request by vaibhavsingh97**
Fixes #

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

Review link: https://infinite-tundra-84847.herokuapp.com/apidocs/

### CODE REVIEW

**Comment by yashLadha** [comment]
image

Response from the server

image


As you can see is not allowing the server to accept __POST__ request 

image

**Comment by vaibhavsingh97** [comment]
@yashLadha @djmgit I tried to push to Heroku and I am getting some database error on Heroku. I will retry and upload full functional API

File: Procfile [hunk 1]
```diff
@@ -1 +1 @@
-web: gunicorn --pythonpath api/ run:app
+web: gunicorn api.run:app
```
Comment (ParthS007) [inline Procfile]: Why Procfile changed ?

**Comment by gabru-md** [comment]
Since this is his second chance and he is stil lunable to provide a working PR, therefore I would like to take this issue up. I had it already mentioned in the issue that I created for the invalid PR of his.

**Comment by yashLadha** [comment]
As I previously mentioned with all the proofs that it won't work @vaibhavsingh97 . Can you please reply to the queries mentioned above in review?

**Comment by gabru-md** [comment]
Please close this PR then. Thanks!

**Comment by gabru-md** [comment]
@djmgit @ParthS007 please close this

**Pull request closed without merging**